docs/design/manipulation/features.md

```diff
@@ -0,0 +1,11 @@
+ Selected features and their suggested importance
+
+| Feature | Description | Importance |
+| ------- | ----------- | ---------- |
+| Modular and configurable design | <ul><li> Highly customizable – can support variety of robots/manipulators </li><li> Modular – easy to replace implementations of parts with alternatives </li><li> Good defaults – less steep learning curve </li></ul> | Crucial |
+| Core dynamics | <ul><li>Realistic movements for manipulation operations</li><li>Consider all the physics parameters involved in the manipulation operations</li></ul> | Crucial |
+| Manipulator configuration |<ul><li>Kinematics of the manipulator, number of actuators, end effectors, etc.</li><li>Use semantic groups to define the different manipulator parts such as arm and end effector</li></ul> | Crucial |
+| Motion planning | <ul><li> Ability to request a motion planning for the manipulator given different kinds of inputs. </li><li> Generate information from O3DE simulation about the state of the movable parts </li><li>Handle self collisions among the manipulator parts</li></ul> | Crucial |
+| Motion execution |<ul><li> Successfully apply techniques of motion execution (controllers) to a O3DE simulation</li><li> Capacity to interact with third party libraries providing controllers </li><li> Interface to develop native controllers inside O3DE </li><li> Receive and execute trajectories generated by third party planners</li></ul> | Crucial |
+| Input system | <ul><li>Basic input for an end effector in the form of a 6D Pose (position + orientation).</li><li>Consider other inputs based only in the position or orientation<li><li>Support goals based on the position/angle of each joint in the manipulator</li></ul> | Crucial |
+| Multirobot collaborative manipulation | <ul><li>Use multiple robot for manipulation</li></ul> | Optional |
```

Comment:
For manipulation, the realism of the contact physics is a key distinguishing factor differentiating between "toy" simulation and actual applicability in real-world use cases, e.g. for a majority of pick & place application. 

```suggestion
| Multirobot collaborative manipulation | <ul><li>Use multiple robot for manipulation</li></ul> | Optional |
| High-fidelity physics model | <ul><li>Support for multiple contact points between bodies</li><li>Soft-contact model</li><li>Soft-body and deformable object physics simulation</li><li>Efficient and stable solver for articulate-chain systems</li></ul> | Optional |
```

While 90% of this feature actually probably need to be / should be implemented within O3DE, this is an essential capability for a manipulation simulator. Could e.g. be realized through upgrading the PhysX Gem to PhysX 5.1.